tck/bridge-tck-main-portlet/src/main/java/com/liferay/faces/bridge/tck/tests/chapter_5/section_5_2/Tests.java

```diff
@@ -602,6 +602,41 @@ public PhaseId getPhaseId() {
 		return PhaseId.ANY_PHASE;
 	}
 
+	// Test is SingleRequest -- Render/Action
+	// Test #5.33 --
+	@BridgeTest(test = "headerPhaseListenerTest")
+	public String headerPhaseListenerTest(TestRunnerBean testRunner) {
```

Comment:
@ngriffin7a, sounds like a good idea. Would you like me to do that before or after the PR is merged?